tabular/src/autogluon/tabular/configs/config_helper.py

```diff
@@ -0,0 +1,357 @@
+        return self
+
+    def enable_vision_features(self, value: bool = True) -> FeatureGeneratorBuilder:
+        """
+        [Experimental]
+        Whether to keep 'object' features identified as 'image_path' special type. Features of this form should have a string path to an image file as their value.
+        Only vision models can leverage these features, and these features will not be treated as categorical.
+        Note: 'image_path' features will not be automatically inferred. These features must be explicitly specified as such in a custom FeatureMetadata object.
+        Note: It is recommended that the string paths use absolute paths rather than relative, as it will likely be more stable.
+        """
+        self.config['enable_vision_features'] = value
+        return self
+
+    def vectorizer(self, value: BaseEstimator) -> FeatureGeneratorBuilder:
+        """
+        sklearn CountVectorizer object to use in TextNgramFeatureGenerator.
+        Only used if `enable_text_ngram_features=True`.
+        """
+        self.config['vectorizer'] = value
+        return self
+
+    def text_ngram_params(self, value: bool = True) -> FeatureGeneratorBuilder:
+        """
+        Appends TextNgramFeatureGenerator(vectorizer=vectorizer, text_ngram_params) to the generator group. See text_ngram.py for valid parameters.
+        """
+        self.config['text_ngram_params'] = value
+        return self
+
+    def build(self) -> ConfigBuilder:
+        self.parent.config['feature_generator'] = AutoMLPipelineFeatureGenerator(**self.config)
+        return self.parent
+
+
+class ConfigBuilder:
+    def __init__(self):
+        self.config = {}
+
+    def presets(self, presets: Union[str, list]) -> ConfigBuilder:
+        """
+        List of preset configurations for various arguments in `fit()`. Can significantly impact predictive accuracy, memory-footprint, and inference latency of trained models, and various other properties of the returned `predictor`.
+        It is recommended to specify presets and avoid specifying most other `fit()` arguments or model hyperparameters prior to becoming familiar with AutoGluon.
+        Available Presets: ['best_quality', 'high_quality_fast_inference_only_refit', 'good_quality_faster_inference_only_refit', 'medium_quality_faster_train', 'optimize_for_deployment', 'ignore_text']
+        It is recommended to only use one `quality` based preset in a given call to `fit()` as they alter many of the same arguments and are not compatible with each-other.
+        If there is an overlap in presets keys, the latter presets will override the earlier ones.
+        """
+        valid_keys = list(tabular_presets_dict.keys())
+        if not isinstance(presets, list):
+            presets = [presets]
+
+        unknown_keys = [k for k in presets if k not in valid_keys]
+        assert len(unknown_keys) == 0, f'The following preset are not recognized: {unknown_keys} - use one of the valid presets: {valid_keys}'
```

Comment:
The following preset are not -> The following presets are not

Reply:
fixed